src/controllers/legacy/researchContent.js

```diff
@@ -11,7 +11,7 @@ import FileStorage from './../../storage';
 import crypto from 'crypto';
 import slug from 'limax';
 import * as blockchainService from './../../utils/blockchain';
-import ResearchGroupService from './../../services/legacy/researchGroup';
+import TeamDtoService from './../../services/impl/read/TeamDtoService';
```

Comment:
Please add `TeamDtoService` to `/services/index.js` and do imports from the index file. It is related to all usages of services